cg/exc.py

```diff
@@ -192,6 +192,14 @@ class SampleSheetError(CgError):
     """Raised when something is wrong with the sample sheet."""
 
 
+class SampleSheetContentError(CgError):
+    """Raised when something is wrong with the sample sheet content."""
+
+
+class SampleSheetFormatError(CgError):
+    """Raised when something is wrong with the sample sheet format."""
```

Comment:
You could create a parent class `SampleSheetError` if you need to catch both of these

Reply:
No, I don't really need to catch both